logfile

```diff
@@ -0,0 +1 @@
+
```

Comment:
just added logfile to my global gitignore, sorry for including.  possible to delete file?